application.go

```diff
@@ -94,7 +100,19 @@ func ApplicationFromConfig(isClient bool, file string) (Application, error) {
 		}
 
 		if !foundTarget {
-			return errMsg("No targets specified")
+			return errMsg("No single targets specified (need at least one)")
```

Comment:
"No server targets defined"

or s/server/regular/  "single" is ambiguous.

or "No 'Targets' entries defined" - probably the clearest